src/Inciter/Partitioner.h

```diff
@@ -1270,6 +1276,7 @@ class Partitioner : public CBase_Partitioner< HostProxy,
     }
     //! \brief Compute standard deviation of the communication cost of merging
     //!   the linear system
+    //! \param[in] host Host to signal to
```

Comment:
I'm not sure if I understand your question. The reason why this is done this way, is explained starting at line [1209](https://github.com/quinoacomputing/quinoa/blob/mms/src/Inciter/Partitioner.h#L1209).